conf.py

```diff
@@ -166,6 +166,12 @@
 # sphinx-notfound-page
 notfound_urls_prefix = "/"
 
+# prolog and epilogs
+rst_prolog = """
+.. |draft| replace:: **Draft** of Python Contributor's Guide
+.. |purpose| replace:: See more info about the contributor guide at https://discuss.python.org/t/refactoring-the-devguide-into-a-contribution-guide/63409. Questions? Ask the docs editorial board.
```

Comment:
Link to where the docs EB can be found?

Reply:
Done [adb60bc](https://github.com/python/devguide/pull/1388/commits/adb60bc0fd07c4cabfe0b0497ad45d45f4fe7246)